packages/block-editor/src/components/inner-blocks/default-block-appender.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I'm not being able to use this component. Changing the column block to:
```
			<InnerBlocks
				templateLock={ false }
				renderAppender={ () => (
					<InnerBlocks.ButtonAppender />
				) }
			/>
```
Does not seem to render any appender. I'm missing something?

Reply:
Not sure what was happening for you (a regression in an earlier commit?) but I've tested with this inside `column.js`

```jsx
<InnerBlocks
	templateLock={ false }
	renderAppender={ () => (
		<InnerBlocks.ButtonBlockAppender />
	) }
/>
```

...and it works perfectly.

![Screen Shot 2019-04-11 at 11 18 35](https://user-images.githubusercontent.com/444434/55949956-b32ec480-5c4b-11e9-8c9a-3335f7cc12ed.png)